src/Plugins/SimplnxCore/wrapping/python/simplnxpy.cpp

```diff
@@ -506,6 +506,55 @@ PYBIND11_MODULE(simplnx, mod)
       },
       "Convert numpy dtype to simplnx DataType", "dtype"_a);
 
+  mod.def(
+      "convert_np_dtype_to_numerictype",
```

Comment:
```suggestion
      "convert_np_dtype_to_numeric_type",
```
For consistency with the other function name.